homepageapp/views.py

```diff
@@ -0,0 +1,190 @@
+from datetime import date
+from django.contrib import messages
+from django.contrib.auth import authenticate, login, logout
+from django.contrib.auth.models import User
+from django.contrib.sites.shortcuts import get_current_site
+from django.core.mail import EmailMessage
+from django.core.mail import send_mail
+from django.shortcuts import render, redirect
+from django.template.loader import render_to_string
+from django.utils.encoding import force_bytes, force_str
+from django.utils.http import urlsafe_base64_encode, urlsafe_base64_decode
+from EquiTimeProject import settings
+from .models import PhoneModel
+from .tokens import generate_token
+import phonenumbers
+
+# Create your views here.
+
+def home_view(request):
+    today = date.today()
+    current_day = today.strftime("%A")
+
+
+    return render(
+        request,
+        'homepageapp/home.html',
+        context={
+            'today': today,
+            'current_day': current_day,
+        }
+    )
+
+
+def register_view(request):
+    if request.method == "POST":
+        username = request.POST.get('username')
+        fname = request.POST.get('fname')
+        lname = request.POST.get('lname')
+        email = request.POST.get('email')
+        pass1 = request.POST.get('pass1')
+        pass2 = request.POST.get('pass2')
+
+        if User.objects.filter(username=username) and User.is_active:
```

Comment:
This should be rewritten to forms:
https://docs.djangoproject.com/en/4.2/topics/forms/
This will give you all this code for free. And much more.